Resources/Prototypes/Entities/Objects/Tools/tools.yml

```diff
@@ -175,7 +175,7 @@
   - type: MeleeWeapon
     damage:
       types:
-        Blunt: 8 #free man
+        Blunt: 8
         Structural: 3
```

Comment:
Just get rid of this since it's the same as parent

Reply:
pro